Extensions/TextObject/textruntimeobject.ts

```diff
@@ -578,15 +744,6 @@ namespace gdjs {
       this._renderer.updateStyle();
     }
 
-    /**
-     * Show the shadow of the text object.
-     * @param enable true to show the shadow, false to hide it
-     */
-    showShadow(enable: boolean): void {
```

Comment:
Should this be kept for retro-compatibility purposes?

Reply:
I only moved it.